packages/itwin/measure-tools/src/measurements/AreaMeasurement.ts

```diff
@@ -42,6 +43,7 @@ import { MeasureTools } from "../MeasureTools";
  */
 export interface AreaMeasurementProps extends MeasurementProps {
   polygonPoints: XYZProps[];
+  drawingMetaData?: DrawingMetaData;
```

Comment:
Move `drawingMetaData` up to `MeasurementProps`

Reply:
Opps, I had forgotten to remove it